### PR TITLE
Land Claim Flag

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
It's just the station charter but now a flag

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
Doot doot doot!

Is this item reusable or a 1 time use kind of thing? It'd be nice if miners could purchase a 1 time land claim for building their own pads for mining points.

**Comment by Repukan** [comment]
Can we combine a combat knife to this to make it a flag spear so I can stab it through a lizard into the NT claimed land.

**Comment by XDTM** [comment]
It only works on a corpse if you do that, no takebacks

**Comment by Repukan** [comment]
The blood of libardz must run for gorborate not land claims. Phoebe lutsu is a roelplayer and not my MD ok :DDD.  Balids and kills not rp and chill  :DDD

**Comment by KorPhaeron** [comment]
It's force 15 so you could beat someone to death with it relatively easily actually

I thought about making it higher force but giving it a slowdown as well

**Comment by kevinz000** [comment]
capture the flag

**Comment by octareenroon91** [comment]
>It'd be nice if miners could purchase a 1 time land claim for building their own pads on lavaland for mining points.

Can't open ground on lavaland be freely assigned to new areas by anyone?

**Comment by Cheridan** [comment]
yeah i thought there was a hud button for this why is it an item

**Comment by octareenroon91** [comment]
This PR is for a station-name designator, not an `areaeditor`

**Comment by FantasticFwoosh** [comment]
@octareenroon91 true, but we are just discussing the outside possibilities such a similar conceptual item could be.

**Comment by octareenroon91** [comment]
Cheridan asked

File: code/game/objects/items/charter.dm [hunk 1]
```diff
@@ -94,4 +95,27 @@
...
+
+/obj/item/station_charter/flag/rename_station(designation, uname, ureal_name, ukey)
+	set_station_name(designation)
+	minor_announce("[ureal_name] has designated the planet as [station_name()]", "Captain's Banner", 0)
+	log_game("[ukey] has renamed the planet as [station_name()].")
+	name = "banner of [station_name()]"
+	desc = "The banner bears the official coat of arms of Nanotrasen, signifying that [station_name()] has been claimed by Captain [uname] in the name of the company."
+	SSblackbox.set_details("station_renames","[station_name()]")
+	if(!unlimited_uses)
+		used = TRUE
```
Comment (Cyberboss) [inline code/game/objects/items/charter.dm]: These three lines should be in a common proc

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
I feel like this should be repurposed

basically make it make a 20x20 area around you marked as yours with whatever name you set, so you can rush out to stake your claim and shit

also then territory wars and shit occur and 
d y n a m i c
r o l e p l a y
occurs

**Comment by FantasticFwoosh** [comment]
@Iamgoofball it should be a core mechanic of the nations roundtype, you must capture & supplant your own flag upon protected pedestals in each of the departments or something, successfully take it over and remaining people in that department nationflip.

**Pull request merged**